index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <section>
+        <header>
+           <div class="logo">
+            <span>My Logo</a></span>
+           </div>
+           <div class="hamburger">
+            <div class="bar"></div>
+            
+           </div> 
+           <nav>
+            <ul class="nav-list">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+         </nav>
```

Comment:
- [ ] kindly wrap your hamburger button inside a `<nav>` tag 🙏 , its for semantics reasons, and the right position for convention and user experience for mobile devices